go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/onsi/gomega v1.19.0
 	github.com/openyurtio/api v0.0.0-20220907024010-e5bfc9cc1b4b
 	github.com/pkg/errors v0.9.1
+	github.com/sirupsen/logrus v1.9.0
```

Comment:
`yurt-edgex-manager` does not use `logrus` library. Try to delete it, then run `go mod tidy`.